apps/server/src/modules/files-storage/uc/files-storage.uc.ts

```diff
@@ -104,8 +103,29 @@ export class FilesStorageUC {
 
 			return result;
 		} catch (error) {
-			this.logger.warn(`could not find file by url: ${params.url}`, error);
-			throw new NotFoundException(ErrorType.FILE_NOT_FOUND);
+			this.logger.warn({
+				message: 'could not find file by url',
+				url: params.url,
+				error: error as Error,
+			});
+			throw new NotFoundException('FILE_NOT_FOUND');
+		}
+	}
+
+	private async uploadFile(userId: EntityId, params: FileRecordParams, fileDescription: IFile) {
```

Comment:
looks like you have a merge conflict here, this function does not exist on main anymore

Reply:
✔️